sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -113,6 +113,15 @@ object DateTimeUtils extends SparkDateTimeUtils {
     getLocalDateTime(micros, zoneId).getMinute
   }
 
+  /**
+   * Returns the minute value of a given TIME (TimeType) value.
+   * The `ZoneId` parameter is ignored because TIME values do not require time zones.
+   * It is included only to satisfy the `GetTimeField` function signature.
+   */
+  def getMinutesFromTime(micros: Long, zoneId: ZoneId): Int = {
+    SparkDateTimeUtils.microsToLocalTime(micros).getMinute
```

Comment:
`microsToLocalTime` should be in the scope:
```suggestion
    microsToLocalTime(micros).getMinute
```